crates/nu-command/tests/commands/parse.rs

```diff
@@ -211,4 +211,19 @@ mod regex {
 
         assert_eq!(actual.out, "2");
     }
+
+    #[test]
+    fn parse_handles_external_stream_chunking() {
+        Playground::setup("parse_test_streaming_1", |dirs, _sandbox| {
+            let actual = nu!(
+                cwd: dirs.test(), pipeline(
+                r#"
+                    "abcdefghijklmnopqrstuvwxyz" * 1000 | save --force data.txt;
+                    open data.txt | parse --regex "(abcdefghijklmnopqrstuvwxyz)" | length
+                "#
+            ));
+
+            assert_eq!(actual.out, "1000");
+        })
+    }
```

Comment:
maybe we can have a simpler test like this?
```suggestion
    #[test]
    fn parse_handles_external_stream_chunking() {
        let actual = nu!(
            pipeline(
            r#"
                "abcdefghijklmnopqrstuvwxyz" * 1000 | save --force data.txt;
                open data.txt | parse --regex "(abcdefghijklmnopqrstuvwxyz)" | length
            "#
        ));

        assert_eq!(actual.out, "1000");
    }
```

not sure if `save` requires to be in a playground, though, i might be wrong on that one :wink:

Reply:
The CWD of the test and `nu!` macro seems to be the source folder, so the Playground (or something else that gives me a reasonable dir to write to) seems to be needed.

Reply:
Thanks @panicbit for investigating.

Reply:
@panicbit 
that's interesting to know, thanks :pray: 

TL;DR: when writing a file to disk in the tests, a playground is needed :ok_hand: